nc/source/input/Input.cpp

```diff
@@ -1,3 +1,4 @@
+#include "Input.h"
```

Comment:
I don't think anything actually changed in this file. Not sure what is happening with the diff.